pom.xml

```diff
@@ -88,7 +88,7 @@
         <jackson1.version>1.9.13</jackson1.version>
         <jackson2.version>2.12.1</jackson2.version>
         <lucene-solr.version>8.6.2</lucene-solr.version>
-        <elasticsearch.version>7.10.2</elasticsearch.version>
+        <elasticsearch.version>7.11.0</elasticsearch.version>
```

Comment:
This is only used by elasticsearch-rest-client. We should bump the version used by Elasticsearch server (which is used in tests) as well.

Further more, I wonder if we should test compatibility using older version of ES client as well, since we are still supporting 6.x, and there is no guarantee that ES 7 client is fully compatible with ES 6 server.

Reply:
Why do need to use older client? 
If I supporting both client version would require to create solution such as we already have for hbase.

Tests are executed against older version of the server https://github.com/JanusGraph/janusgraph/pull/2453/checks?check_run_id=1902862139.

Reply:
We should bump the version used by Elasticsearch server (which is used in tests) as well. That would be a good idea.

Reply:
> Why do need to use older client

This is arguable and might be very rare, but here is a hypothetical use case: the user is using the ES6 server and wants to force the usage of the ES6 client as well. They could do so by excluding ES7 client dependency introduced by janusgraph and explicitly introducing ES6 client dependency. It could happen that some methods we use in our codebase do not exist in the earlier version of the client.

https://docs.janusgraph.org/changelog/ claims janusgraph 0.5.2 supports Elasticsearch 6.0.1. This is fine, but let's just assume for a moment that the doc claims janusgraph supports Elasticsearch 6.0.0. Then the user might want to use ES 6.0.0 client + JanusGraph 0.5.2 + ES 6.0.0 server. This would not work because JanusGraph 0.5.2 codebase uses some ES client methods which are only available since ES 6.0.1. The user could still use ES 7.x client (which is a dependency of JanusGraph 0.5.2) + JanusGraph 0.5.2 + ES 6.0.0 server because this is already tested by our CI, but they might wonder why ES 6.0.0 client does not work, given ES 6.0.0 support is documented.

The above case might be very rare though, since users are supposed to use the ES client version that comes as part of janusgraph dependency. To avoid it happening, we need to be careful when changing the code of ES client usage.